pkg/lib/service.js

```diff
@@ -47,11 +47,15 @@ import cockpit from "cockpit";
  * service.
  *
  * - proxy.unit
- * - proxy.service
+ * - proxy.details
  *
- * The raw org.freedesktop.systemd1.Unit and Service D-Bus
+ * The raw org.freedesktop.systemd1.Unit and type-specific D-Bus
```

Comment:
This would be useful to have on master as well.

Reply:
Yes: #12728